### DISCUSSION TITLE
Ability to search panel content with Control+F or Command+F when accordions are collapsed

### DISCUSSION THREAD

**Discussion by todor0v** [payloadcms/payload#6934] [Feature Requests & Ideas]
When the accordions in the panel created by group or array fields are collapsed, it becomes impossible to use the default browser search functionality ("Control+F" or "Command+F" on a Mac) to locate content. By default, all the accordions are expanded, but this changes over time as more users interact with the panel. It would be ideal if text could be found even in collapsed fields, and even better if the parent accordion expanded automatically to reveal the information. If this is not possible or is difficult to implement, I would appreciate a search feature for a specific page (single collection or global item).

**Comment by yandodov** [comment]
Yes, this is something very important for our editors. Our pages are quite long and they use Cmd+F habitually to find what they look for.

My guess is that the hidden accordions are completely out of the DOM due to the way they're implemented in React. Perhaps one solution is to hide them with `display: none`, **while keeping them in the DOM**, so they can be found via search, then expand them if the user lands on them.

This would make the content management experience way easier for our editors. I hope it gets considered!
